src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteModel.js

```diff
@@ -291,6 +291,25 @@ export default class StandardRouteModel extends BaseModel {
     }
 
     /**
+     * Returns an array of unique fix names used by any portion of the procedure
+     *
+     * @for StandardRouteModel
+     * @method getAllFixNamesInUse
+     * @return {array<string>} ['fixname', 'fixname', 'fixname', ...]
+     */
+    getAllFixNamesInUse() {
+        const allFixNames = [
+            ...this._entryCollection.getAllFixNamesInUse(),
+            ...this._bodySegmentModel.getAllFixNamesInUse(),
+            ...this._exitCollection.getAllFixNamesInUse(),
+            ...this.draw.reduce((list, fix) => list.concat(fix)).map((fixName) => fixName.replace('*', ''))
```

Comment:
There is a lot going on and n this one line.  This should be set to a var instead of inline like this.

Reply:
changed to:
```
const drawFixNames = this.draw.reduce((list, fix) => list.concat(fix))
    .map((fixName) => fixName.replace('*', ''));

const allFixNames = [
    ...this._entryCollection.getAllFixNamesInUse(),
    ...this._bodySegmentModel.getAllFixNamesInUse(),
    ...this._exitCollection.getAllFixNamesInUse(),
    ...drawFixNames
];
```